source/windows/launching_window.py

```diff
@@ -326,6 +324,11 @@ def search_finished(self):
                 if self.version_query is None:
                     self.update_query_boxes(vsq)
 
+            if self.save_current_query_button is not None:
+                self.save_current_query_button.setText(
+                    f"Save Blender {self.version_query.major}.{self.version_query.minor}.{self.version_query.patch} as default version for this file"
```

Comment:
The button currently saves the query for minor version changes, for example:
```json
{
    "4.3.0": "4.^.^",
    "4.0.0": "4.3.^",
}
```
In this case, I think the original text makes more sense. I think it would get messy fast if we were to save a different query for every _individual_ blendfile

Also, maybe update the button when the query changes

> Make the button more meaningful for lambda users

Lambda users?